tests/functional/aws-node-sdk/lib/utility/makeRequest.js

```diff
@@ -0,0 +1,87 @@
+import http from 'http';
```

Comment:
it looks like I might not need to update Integration, it just runs ft_test which runs all the tests in the S3 directory recursively? 
https://github.com/scality/Integration/blob/master/genericStaas/env/run_s3.bash
I'll run an end-to-end and see if my tests are run (after I fix some tests)